src/components/LoanCards/NewHomePageLoanCard.vue

```diff
@@ -249,6 +269,21 @@ export default {
 		loanAmount() {
 			return this.loan?.loanAmount;
 		},
+		dedications() {
+			const dedications = this.loan?.dedications?.values ?? [];
+			const dedicationToKiva = this.loan?.dedicationToKiva ?? false;
+			if (dedicationToKiva) {
+				dedications.push({
+					recipientName: 'Kiva',
+				});
+			}
+			return dedications;
+		},
+		dedicationsCopy() {
+			return this.dedications.length > 1
+				? 'Multiple recipients'
+				: this.dedications[0].recipientName;
```

Comment:
Looks like we still need to check length here, since `dedications` can be empty.

Reply:
If `dedications` is empty the section is not rendered in line 125 so this computed prop is not called.